unitTests/ui/FacetSettingsTest.ts

```diff
@@ -18,6 +18,12 @@ export function FacetSettingsTest() {
       facetSettings.build();
     }
 
+    const waitForPopperJS = () => {
+      return new Promise((resolve, reject) => {
```

Comment:
Since we are not using the `reject`, this could become:
`return new Promise(resolve => setTimeout(() => resolve(true), 100))`